coda-prototype/initUI.js

```diff
@@ -660,12 +660,17 @@ function initUI(dataset) {
                 $(".tableFloatingHeaderOriginal").hide();
                 failAlert.show();
 
-                if (parseErrors.length > 100) { // only report first 100 wrong lines
-                    parseErrors = parseErrors.slice(0, 100);
+                if (error.name === "ParseError") {
+                    console.log("ERROR: Cannot parse scheme file");
+                    if (parseErrors.length > 100) { // only report first 100 wrong lines
+                        parseErrors = parseErrors.slice(0, 100);
+                    }
+                    console.log(JSON.stringify(parseErrors));
+                } else if (error.name === "CodeConsistencyError") {
+                    console.log("ERROR: Uploaded code scheme has multiple ids.");
+                } else {
+                    console.log("ERROR: An unknown error occurred");
```

Comment:
Can we log the error in this case?

Reply:
Yes. This will be done in branch io-refactoring-deduplicate-alerts, because then we can easily display an error message to the user.